spec/components/guardian_spec.rb

```diff
@@ -2692,6 +2692,35 @@
     end
   end
 
+  describe 'can_use_flair_group?' do
+    fab!(:group) { Fabricate(:group, title: 'Groupie', flair_icon: 'icon') }
+
+    it 'is false without a logged in user' do
+      expect(Guardian.new(nil).can_use_flair_group?(user)).to be_falsey
```

Comment:
Is there a reason to use `be_falsey` here since the method will always return a boolean? `be_falsey` is only useful when `false` or `nil` can be returned.

Reply:
`be_falsey` and `be_truthy` are used throughout the whole file.

Reply:
O icic. Consistency is fine but do take note that some of the code in our codebase has been written a long time ago so it may not necessarily make sense in today's context. Anyway, it isn't a big problem here so I'll resolve this :)